cmd/ci-reporter/cmd/root.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+	"os"
+
+	"github.com/google/go-github/v34/github"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"golang.org/x/oauth2"
+	"sigs.k8s.io/release-utils/env"
+)
+
+var rootCmd = &cobra.Command{
+	Use:    "reporter",
+	Short:  "Github and Testgrid report generator",
+	Long:   "CI-Signal reporter that generates github and testgrid reports.",
+	PreRun: setGithubConfig,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return RunReport(*cfg)
+	},
+}
+
+// Execute executes the ci-reporter root command.
+func Execute() error {
+	return rootCmd.Execute()
+}
```

Comment:
Does anyone know if we still need the `Execute` function when we don't have subcommands? Wouldn't the 

```
	RunE: func(cmd *cobra.Command, args []string) error {
```

on line 35 be called automatically if there is just one single command?

Reply:
cobra convention